source/reporting/index.html.md.erb

```diff
@@ -0,0 +1,113 @@
+
+## Reporting via the GOV.UK Pay API 
+
+Before reading this section, you should read the section describing [Making
+payments](/making_payments). 
+
+You can use the GOV.UK Pay API to export data automatically into your finance
+systems. This requires you to do some development, but could remove 
+routine manual operations from your workflow and lead to significant cost
+savings for your organisation. Before you do this, you should have a datastore 
+where you are already recording payments. 
+
+You can use the GOV.UK Pay API to:
+
+* get information about single payments
+* generate a list of payments 
+
+## Get information about single payments
+
+### Check the status of a single payment
+
+`GET /v1/payments/paymentId`
+
+1. Sign in to <a href="https://gds-payments.gelato.io/api-explorer/"
+   target="blank">the API Explorer</a> (link opens in new window).  
+2. Under __Resource__, select __{Payment Id}. Under __Action__, select <a
+   href="https://gds-payments.gelato.io/api-explorer/gov-uk-pay-api/versions/1.0.2/general/"
+   target="blank">__Find payment by ID__</a> (link opens in new window). 
+   
+For a valid request, the JSON response body contains a `"state"` field. 
+
+For example, for a successful payment:
+
+```
+  "state": {
+    "status": "success",
+    "finished": true
+    },
+```
+
+### See information users have entered for a single payment
+
+`GET /v1/payments/paymentId`
+
+1. Sign in to <a href="https://gds-payments.gelato.io/api-explorer/"
+   target="blank">the API Explorer</a> (link opens in new window).  
+2. Under __Resource__, select __{Payment Id}. Under __Action__, select <a
+   href="https://gds-payments.gelato.io/api-explorer/gov-uk-pay-api/versions/1.0.2/general/"
+   target="blank">__Find payment by ID__</a> (link opens in new window). 
+
+For a valid request, the JSON response body contains the `"card_details"`
+section. This shows information that users enter when making a payment. 
+
+For example:
+
+```
+  "card_details": {
+    "last_digits_card_number": "1111",
+    "first_digits_card_number": "100002",
+    "cardholder_name": "Sherlock Holmes",
+    "expiry_date": "12/21",
+    "billing_address": {
+      "line1": "221 Baker Street",
+      "line2": "Flat b",
+      "postcode": "NW16XE",
+      "city": "London",
+      "country": "GB"
+    },
+```
+
+## Generate a list of payments (search payments) 
+
+`GET /v1/payments`
+
+1. Sign in to <a href="https://gds-payments.gelato.io/api-explorer/"
+   target="blank">the API Explorer</a> (link opens in new window).  
+2. Under __Resource__, select __General__. Under __Action__, select <a
+   href="https://gds-payments.gelato.io/api-explorer/gov-uk-pay-api/versions/1.0.2/general/"
+   target="blank">__Search payments__</a> (link opens in new window). 
+
+You can search payments using the following criteria:
+
+* `reference`
+* `email`
+* `state`
+* `card_brand`
+* `from_date`
+* `to_date`
+* `page`
+* `display_size`
```

Comment:
There are more than these so maybe the phrasing that introduces the list should state that they are examples of what can be searched for.

The `page` and `display_size` aren’t really criteria so probably need to explained separately (`display_size` is how many results to show at once, `page` is the page of results you want — so if `display_size` is `500` and `page` is `2` and the criteria match 1,543 payments, matching payments 501 through 1000 will be returned).

Might also be worth explaining `from_date` and `to_date` separately (technically a filter but not usually used in the same way as the others). We should state that these dates relate to when the payments were created too.

Reply:
These are the criteria I copied from the API explorer. Do we have an exhaustive list documented anywhere or is there a way to probe the API to find what can be searched? If so I wonder if it would be better to omit examples altogether for now.

Reply:
Currently we have `"reference"`, `"email"`, `"state"`, `"agreement_id"` (only applicable for Direct Debit, so we might want to exclude this one), `"card_brand"`, `"first_digits_card_number"`, `"last_digits_card_number"`, `"cardholder_name"`, `"from_date"`, and `"to_date"`